lib/modules/commentQuickCollapse.js

```diff
@@ -58,7 +58,7 @@ module.options = {
 	hideCollapseButton: {
 		dependsOn: 'toggleCommentsOnClickLeftEdge',
 		type: 'boolean',
-		value: true,
+		value: false,
```

Comment:
Since you added a check to see if the whole module was disabled I think this should be on by default now. The feature doesn't look that good with this enabled.